sdk/core/azure-core-management/src/main/java/com/azure/core/management/Resource.java

```diff
@@ -47,7 +47,7 @@ public Resource setLocation(String location) {
      *
      * @return the tags value
      */
-    public Map<String, String> getTags() {
+    public Map<String, String> tags() {
```

Comment:
in track1, it is `getTags`

Reply:
Fixed according to track 1 style. Though it's strange they put getTags/withTags together.

Reply:
any context on why track1 is getTags(), but not tags? If no specific reason, this is good time for us to fix it for aligment.